invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -100,6 +100,20 @@ export class RDMDepositForm extends Component {
     const { record, files, permissions, preselectedCommunity, filesLocked } =
       this.props;
     const customFieldsUI = this.config.custom_fields.ui;
+    const filteredCustomFieldsUI = customFieldsUI.map((section) => {
```

Comment:
changes in this file are not needed in combination with my PRs, could you please remove it?